tsrc/test/cli/test_sync.py

```diff
@@ -191,6 +192,73 @@ def test_sync_with_force(
     ).read_text() == "2", "foo should have been reset to the latest tag"
 
 
+def add_repo_unstaged(name: str, git_server: GitServer, workspace_path: Path) -> None:
+    """Add a repo to the manifest without staging this change."""
+    repo_config = {"url": git_server.get_url(name), "dest": name}
+    manifest_data = git_server.manifest.data.copy()
+    manifest_data["repos"].append(repo_config)
+    manifest_path = workspace_path / ".tsrc" / "manifest" / "manifest.yml"
+    with open(manifest_path, "w") as manifest:
+        to_write = ruamel.yaml.dump(manifest_data)
+        assert to_write
+        manifest.write(to_write)
+
+
+def test_sync_without_no_update_manifest_flag_discards_changes(
```

Comment:
See why I don't like double negatives :P You can rename this function to be:

`def test_sync_discards_local_manifest_changes()`